src/app/routes/utils/regex/utils/index.js

```diff
@@ -47,9 +47,12 @@ const homePageServices = [
   'igbo',
   'indonesia',
   'kyrgyz',
+  'korean',
   'marathi',
   'nepali',
+  'pashto',
   'persian',
+  'persian/afghanistan',
```

Comment:
This is not required here - /persian/afghanistan is not a homepage, but a topic page. The changes to src/app/routes/topic/index.js and to `getIdxPageRegex` below are the code to launch /persian/afghanistan to live. We only need to include korean and pashto in the list of homepage services to migrate here.